src/main/java/mekanism/common/tile/prefab/TileEntityEffectsBlock.java

```diff
@@ -82,10 +84,17 @@ private void updateSound() {
                 return;
             }
 
-            if (activeSound == null || !Minecraft.getMinecraft().getSoundHandler().isSoundPlaying(activeSound)) {
+            // If this machine isn't fully muffled and we don't seem to be playing a sound for it, go ahead and
+            // play it
+            if (!isFullyMuffled() && (activeSound == null || !Minecraft.getMinecraft().getSoundHandler().isSoundPlaying(activeSound))) {
                 activeSound = SoundHandler.startTileSound(soundEvent.getSoundName(), getInitialVolume(), getPos());
-                playSoundCooldown = 20;
             }
+
+            Mekanism.logger.info("Fully muffled: {}", isFullyMuffled());
```

Comment:
lingering debug?

Reply:
I think it was for purposes of when we tested, I just removed it though and the changes seem to work fine.